clang/test/Driver/aarch64-predres.c

```diff
@@ -1,8 +1,9 @@
 // RUN: %clang -### --target=aarch64-none-elf -march=armv8a+predres     %s 2>&1 | FileCheck %s
+// RUN: %clang -### --target=aarch64-none-elf -mcpu=cortex-a520         %s 2>&1 | FileCheck %s
 // CHECK: "-target-feature" "+predres"
 // CHECK-NOT: "-target-feature" "-predres"
 
-// RUN: %clang -### --target=aarch64-none-elf -march=armv8.5a+nopredres %s 2>&1 | FileCheck %s --check-prefix=NOPR
+// RUN: %clang -### --target=aarch64-none-elf -mcpu=cortex-a520+nopredres %s 2>&1 | FileCheck %s --check-prefix=NOPR
```

Comment:
Did you mean to remove the armv8.5-a line here? As above you added the cortex-a520 instead of replacing the armv8.5 line.

Reply:
Because `armv8.5a` doesn't have the `predres` feature by default, adding `+nopredres` doesn't cause it to appear in the `-###` output. `cortex-a520` does have `predres` by default, so it does.